### PR TITLE
add gene symbol to search results table

### PR DESCRIPTION
**Pull request by jyothishnt**
## Description
Add gene symbol to gene search results  table

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2604

## Deployment URL(s)
http://add-gene-symbol.review.ensembl.org

## Views affected
Gene search in species selector

### CODE REVIEW

File: src/shared/components/gene-search-panel/GeneSearchPanel.module.css [hunk 1]
```diff
@@ -94,6 +94,12 @@
   cursor: pointer;
 }
 
+.geneSymbol {
+  font-weight: var(--font-weight-bold);
+  text-transform: uppercase;
```
Comment (azangru) [inline src/shared/components/gene-search-panel/GeneSearchPanel.module.css:99]: Umm, we shouldn't modify the text of the symbol

File: src/shared/components/gene-search-panel/GeneSearchPanel.module.css [hunk 2]
```diff
@@ -94,6 +94,12 @@
...
+  padding-right: 8px;
```
Comment (azangru) [inline src/shared/components/gene-search-panel/GeneSearchPanel.module.css:100]: Curious: do we need to customise the space to the right of the gene symbol? It is not just a regular space between words?

Comment (jyothishnt) [reply src/shared/components/gene-search-panel/GeneSearchPanel.module.css:100]: XD has more gap than a regular space

File: src/shared/components/gene-search-panel/GeneSearchPanel.module.css (hunk 1 above)
Comment (azangru) [reply src/shared/components/gene-search-panel/GeneSearchPanel.module.css:99]: > XD shows all caps though

Some symbols will have all capital letters; some will have lower-case letters (e.g. genes of C. elegans). This cannot be a design decision; it is the way appropriate scientific communities decide to name a gene.

File: src/shared/components/gene-search-panel/GeneSearchPanel.module.css (hunk 2 above)
Comment (azangru) [reply src/shared/components/gene-search-panel/GeneSearchPanel.module.css:99]: Ok. Let's do margin instead of the padding then.

**Pull request merged**